pipeline/ccf.py

```diff
@@ -200,3 +202,18 @@ def get_ontology_regions():
 
     return pd.concat([regions, hexcode], axis=1)
 
+
+def get_ccf_xyz_max():
```

Comment:
if this is slow it might make sense to do this as a computed part of the CCF table after reference volume load

Reply:
it's not too bad, on order of a few seconds, so I think it's fine currently

Reply:
thanks for confirmation